demo/index.html

```diff
@@ -47,7 +25,7 @@ <h1>Local Demo</h1>
 
     <iframe
       id="jupyterlab"
-      src="http://localhost:8888"
+      src="%VITE_DEMO_SRC%"
```

Comment:
With that still work when deployed to GitHub pages?

When trying the build `github-pages` artifact (built on CI) locally, and starting an HTTP server with `python -m http.server`, it gives the following:

![image](https://github.com/user-attachments/assets/9ab563af-5776-4d5b-bf33-2041a1c0b012)

Reply:
I don't think it'll work with the python server, it should work if you run `jlpm start:local` in the `demo` directory, that'll start the Vite server and set the env variables. I'll double check the ghpages deployment on my fork.

Reply:
> With that still work when deployed to GitHub pages?

It does not :melting_face:

Reply:
ok, it should be good now, https://gjmooney.github.io/jupyter-iframe-commands/